src/pydvl/utils/functional.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import inspect
+from functools import partial
+from typing import Callable, Set, Tuple
+
+__all__ = ["unroll_partial_fn_args"]
+
+
+def unroll_partial_fn_args(fun: Callable) -> Set[str]:
+    """
+    Unroll a function that was set by functools.partial.
```

Comment:
What's "unrolling"? From the docstring it's unclear what this does. Please explain and add an example

Reply:
Added docstring.